ext/cuda/bilinearinterpolation.jl

```diff
@@ -6,8 +6,10 @@ import ..ClimaInterpolations.BilinearInterpolation:
     get_dims
 
 # CUDA limit per maximum number of logical blocks per grid dimension
-# The first dimension could have a higher limit, but a convervative
+# The first dimension could have a higher limit, but a conservative
 # value is used here.
+# Please table 23 at https://docs.nvidia.com/cuda/cuda-c-programming-guide/#compute-capabilities
+# for the details.
 _get_maxblocksperdim() = 65535
```

Comment:
```suggestion
_get_maxblocksperdim() = CUDA.attribute(CUDA.device(), CUDA.DEVICE_ATTRIBUTE_MAX_GRID_DIM_Y)
```
It looks like we could alternatively use the CUDA attributes. @juliasloan25, what do you think?